src/scenes/storeScene.js

```diff
@@ -0,0 +1,916 @@
+
+        let scene = this;
+        this.nextCardButton.setInteractive().on('pointerdown', () => scene.nextCard() );
+    }
+
+    takeObject(object_name) {
+      // return; // debug
+      
+      console.log('take', object_name);
+      let object = this.children.getByName(object_name);
+      object.depth = 5;
+      
+      let yDistance = this.basket.y - object.y; // between 280 and 1340
+      console.log('yDistance:', yDistance)
+      let target_objects = [object]
+      if(object_name == 'pate_spaghetti01'){
+        // also move the other pack
+        target_objects.push(this.children.getByName('pate_spaghetti02'));
+      }
+      
+      // Move to basket
+      this.tweens.add({
+          targets: target_objects,
+          x: this.basket.x + (Math.random() * 50),
+          y: this.basket.y + 25 + (Math.random() * 25),
+          duration: 150 + (yDistance / 8),
+          ease: 'Quadratic',
+          yoyo: false,
+          loop: 0,
+          onComplete: () => {
+              // this.rature = this.add.image(this.listPositions[object_name], 'rature')
+              // todo: add flag
+              for (const item in this.checklist_done) {
+                if(object_name.indexOf(item) >= 0){
+                  if(this.checklist_done[item].done == false) {
+                    console.log('ajout rature', item)
+                    this.rature = this.add.image(this.checklist_done[item].position_x, this.checklist_done[item].position_y, 'rature')
+                    this.rature.depth = 20;
+                    this.checklist_done[item].done = true;
+                  } else {
+                    console.log('rature deja ajoutée')
+                  }
+                }else{
+                  console.log('pas de', item)
+                }
+              }
+              
+              // Animate basket
+              this.tweens.add({
+                targets: [this.basket, this.basket_front],
+                scale: 1.2,
+                duration: 100,
+                ease: 'Quadratic',
+                yoyo: true,
+                loop: 0
+              })
+          },
+          onCompleteScope: this
+      });
+    }
+
+    create() {
+        this.cameras.main.centerOn(0, 0);
+        this.cameras.main.fadeIn(1000);
+
+        if(this.current_card.isLoaded()) {
+            this.current_card.create();
+            this.addArrow()
+        }
+        
+        this.checklist = this.add.image(-this.cameras.main.width * 0.255, 489, 'liste');
+        this.checklist.depth = 20;
+        
+        
+        this.basket = this.add.image(this.cameras.main.width * 0.24166, 700, 'basket');
+        this.basket.depth = 4;
+        
+        this.basket_front = this.add.image(this.cameras.main.width * 0.24166, 700, 'basket-front');
+        this.basket_front.depth = 25;
+        
+        // Update the saved data
+        // ! Cette scene apparaitra pour deux personnages: independant, etudiant
```

Comment:
This isn't a problem, all you need to do is specify which version you're currently in in the scene_data (`player.setData()` can help you with that)